pkg/stores/partition/listprocessor/processor.go

```diff
@@ -360,3 +394,28 @@ func PaginateList(list []unstructured.Unstructured, p Pagination) ([]unstructure
 	}
 	return list[offset : offset+p.pageSize], pages
 }
+
+func FilterByProjectsAndNamespaces(list []unstructured.Unstructured, projectsOrNamespaces ProjectsOrNamespacesFilter, namespaceCache corecontrollers.NamespaceCache) []unstructured.Unstructured {
+	if len(projectsOrNamespaces.filter) == 0 {
+		return list
+	}
+	result := []unstructured.Unstructured{}
+	for _, obj := range list {
+		namespaceName := obj.GetNamespace()
+		namespace, _ := namespaceCache.Get(namespaceName)
+		if namespace == nil {
```

Comment:
IIRC the go-ism here is that in the case of an `obj, err` return, the presence of an error indicates that the first value is invalid, but not necessarily nil. So in this case you could be getting a corrupt (i.e. non-nil but otherwise invalid object) namespace, but since you are ignoring the error we will keep trying to process it.

TL;DR: Maybe change this (and the preceeding line) to :

```golang
namespace, err := namespaceCache.Get(namespaceName)
if namespace == nil || err != nil {
```